.github/workflows/rubocop.yml

```
@@ -0,0 +1,17 @@
+name: Rubocop Check
+on: [pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.1.2
```

I think you are using 3.2.2?
```suggestion
          ruby-version: 3.2.2
```